mypy.ini

```diff
@@ -24,3 +24,51 @@ warn_unreachable = True
 strict_equality = True
 ignore_missing_imports = True
 
+[mypy-aiven.client.common]
+ignore_errors = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+no_implicit_optional = True
+warn_unused_ignores = True
+warn_no_return = True
+warn_unreachable = True
+strict_equality = True
+ignore_missing_imports = True
+
+[mypy-aiven.client.envdefault]
+ignore_errors = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+no_implicit_optional = True
+warn_unused_ignores = True
+warn_no_return = True
+warn_unreachable = True
+strict_equality = True
+ignore_missing_imports = True
+
+[mypy-aiven.client.session]
+ignore_errors = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+no_implicit_optional = True
+warn_unused_ignores = True
+warn_no_return = True
+warn_unreachable = True
+strict_equality = True
+ignore_missing_imports = True
+
+[mypy-aiven.client.version]
+ignore_errors = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+no_implicit_optional = True
+warn_unused_ignores = True
+warn_no_return = True
+warn_unreachable = True
+strict_equality = True
+ignore_missing_imports = True
```

Comment:
I'd do an inverse of it. Set `ignore_errors = False` for `mypy-aiven.client` and then disable it for sub-modules / sub-packages for which type checking are not enabled yet. This way it will help if somebody will add a new sub-module in a meantime. And also it will be an ultimate goal to only keep `mypy-aiven.client` block at the end.